dasharo-compatibility/usb-hid-and-msc-support.robot

```diff
@@ -66,6 +66,7 @@ USB002.001 USB keyboard detected in FW
     [Tags]    minimal-regression
     Power On
     Enter UEFI Shell
+    Set Prompt For Terminal    Shell>
```

Comment:
I wonder why setting prompt is not part of the keyword entering UEFI shell?

Reply:
I moved it to the `Enter UEFI Shell` kwd https://github.com/Dasharo/open-source-firmware-validation/pull/528/commits/ea62cb11e4de15ab9f5da7e7cada863d73351048